src/sensor/ping.h

```diff
@@ -59,7 +59,46 @@ class Ping : public Sensor
      * @return uint8_t source id
      */
     uint8_t srcId() { return _srcId; }
-    Q_PROPERTY(int srcId READ srcId NOTIFY srcIdUpdate)
+
+    /** TODO: This is not working
+     * @brief Set device id
+     *
+     * @param id
+     */
+    void srcId(int id)
```

Comment:
I think maybe we should refactor this to device_id

srcID is in context of a message. Here we are representing state of a device.

Reply:
oooops, this should not be here.